sdk/storage/azure-storage-queues/vcpkg/vcpkg.json

```
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+{
+  "name": "azure-storage-queues-cpp",
+  "version-semver": "@AZ_LIBRARY_VERSION@",
+  "description": [
+    "Microsoft Azure Storage Queues SDK for C++",
+    "This library provides Azure Storage Queues SDK."
+  ],
+  "homepage": "https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/storage/azure-storage-queues",
+  "license": "MIT",
+  "dependencies": [
+    {
+      "name": "azure-storage-common-cpp",
+      "default-features": false
```

```suggestion
      "default-features": false,
      "version>=": "12.0.0"
```
(this was added just a day ago, and this is how we should do it now. If Queues require updates to Storage-Common and would require greater version, update it here)